src/providers/BackgroundBlurProvider/index.tsx

```diff
@@ -51,7 +49,8 @@ export const BackgroundBlurProvider: FC<React.PropsWithChildren<Props>> = ({
   options,
   children,
 }) => {
-  const logger = useLogger();
+  let logger = useLogger();
+  logger = options?.logger || logger;
```

Comment:
Why not just one line `let logger = options?.logger || useLogger()` ?

Reply:
there's warning for conditionally calling hooks

Reply:
how about `let logger = options?.logger ?? useLogger()` ?

Reply:
same warning